docs/docs/getting-started/first-steps.md

```diff
@@ -164,6 +171,46 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
 
         Supported are all machines from the N2D family. It defaults to `n2d-standard-4` (4 vCPUs, 16 GB RAM), but you can use any other VMs from the same family. Refer to [N2D machine series](https://cloud.google.com/compute/docs/general-purpose-machines#n2d_machines) or run `constellation config instance-types` to get the list of all supported options.
 
+    </tabItem>
+    <tabItem value="aws" label="AWS">
+
+    * **region**: The name of your chosen AWS data center region, e.g., `us-east-2`.
+
+        Constellation OS images are currently replicated to the following regions:
+        * `eu-central-1`
+        * `us-east-2`
+        * `ap-south-1`
+
+        If you require the OS image to be available in another region, [let us know](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&template=feature_request.md&title=Support+new+AWS+image+region:+xx-xxxx-x).
+
+        You can find a list of all [regions in AWS's documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions).
+
+    * **zone**: The name of your chosen AWS data center availability zone, e.g., `us-east-2a`.
+
+        Learn more about [availability zones in AWS's documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-availability-zones).
+
+    * **image**: The ID of the amazon machine image (AMI) the Constellation nodes will use:
+
+        Constellation OS images are available with the following IDs: TODO: Fill in AMI IDs once released.
+
+        | AMI | Region |
+        | - | - |
+        | `ami-0xxxxxxxxxxxxxxxx` | `eu-central-1` |
+        | `ami-0xxxxxxxxxxxxxxxx` | `us-east-2` |
+        | `ami-0xxxxxxxxxxxxxxxx` | `ap-south-1` |
+
+    * **iamProfileControlPlane**: The name of a profile attached to all control plane nodes.
+
+        Use the [provided Terraform script](https://github.com/edgelesssys/constellation/blob/main/hack/terraform/aws/iam) to generate the necessary profile. It will be provided as Terraform output variable: `control_plane_instance_profile`.
```

Comment:
Since there is currently a [discussion on where to put the script](https://github.com/edgelesssys/constellation/pull/478), I'd suggest to pin a commit in the link. Even more so, because when we implement `constellation iam create` we will probably move those files again. 

I'm also fine with linking against the release branch, as the place _should_ be stable.